api/db/seeds/data/campaign-participations-builder.js

```diff
@@ -35,19 +35,29 @@ module.exports = function addCampaignWithParticipations({ databaseBuilder }) {
   const usersNotShared = [users[4], users[5], users[6], users[7], users[8]];
   const usersCompletedShared = [users[0], users[9], users[10], users[11], users[12]];
 
-  const participateToCampaignOfAssessment = (campaignId, user, isShared, validatedSkillsCount = null) => {
+  const participateToCampaignOfAssessment = (campaignId, user, isShared, validatedSkillsCount = null, isImproved = false) => {
     const sharedAt = isShared ? new Date() : null;
     const participantExternalId = user.firstName.toLowerCase() + user.lastName.toLowerCase();
-    return databaseBuilder.factory.buildCampaignParticipation({ campaignId, userId: user.id, participantExternalId, isShared, sharedAt, validatedSkillsCount });
+    return databaseBuilder.factory.buildCampaignParticipation({ campaignId, userId: user.id, participantExternalId, isShared, sharedAt, validatedSkillsCount, isImproved });
   };
 
-  const participateComplexAssessmentCampaign = (campaignId, user, state, isShared) => {
+  const participateComplexAssessmentCampaign = (campaignId, user, state, isShared, isImproved = false) => {
     const { id: userId } = user;
     const validatedSkillsCount = isShared ? 3 : null;
-    const { id: campaignParticipationId } = participateToCampaignOfAssessment(campaignId, user, isShared, validatedSkillsCount);
+    const { id: campaignParticipationId } = participateToCampaignOfAssessment(campaignId, user, isShared, validatedSkillsCount, isImproved);
+
     if (['Stéphan', 'Antoine'].includes(user.firstName)) databaseBuilder.factory.buildBadgeAcquisition({ userId, badgeId: PRO_BASICS_BADGE_ID, campaignParticipationId });
     if (['Jaune', 'Antoine'].includes(user.firstName)) databaseBuilder.factory.buildBadgeAcquisition({ userId, badgeId: PRO_TOOLS_BADGE_ID, campaignParticipationId });
 
+    buildAssessmentAndAnswer(userId, campaignParticipationId, state);
+
+    if (isImproved) {
+      const { id: newCampaignParticipationId } = participateToCampaignOfAssessment(campaignId, user, isShared, validatedSkillsCount * 2);
+      buildAssessmentAndAnswer(userId, newCampaignParticipationId, state);
```

Comment:
faire un +1 au lieu du x2 . ajouter dans buildAssessment and answer l'acquis en plus